jackson/src/test/java/com/baeldung/jackson/enums/JacksonEnumSerializationUnitTest.java

```diff
@@ -19,4 +23,41 @@ public final void givenEnum_whenSerializingJson_thenCorrectRepresentation() thro
         assertThat(dtoAsString, containsString("1609.34"));
     }
 
+    @Test
+    @Ignore
```

Comment:
Why are some many tested Ignored? Maybe we should think about having duplicate enum classes with different annotations so that we can run all the tests at the same time without commenting anything out or using the Ignore annotation.